src/hisp/festim_models/mb_model.py

```diff
@@ -248,23 +252,284 @@ def make_W_mb_model(
+        deuterium_ion_flux: the deuterium ion flux in m^-2 s^-1.
+        tritium_ion_flux: the tritium ion flux in m^-2 s^-1.
+        deuterium_atom_flux: the deuterium atom flux in m^-2 s^-1.
+        tritium_atom_flux: the tritium atom flux in m^-2 s^-1.
+        final_time: the final time in s.
+        folder: the folder to save the results.
+        L: the length of the domain in m.
+
+    Returns:
+        the FESTIM model, the quantities to export.
+    """
+    my_model = CustomProblem()
+
+    ############# Material Parameters #############
+
+    L = L  # m
+    vertices = np.concatenate(  # 1D mesh with extra refinement
+        [
+            np.linspace(0, 30e-9, num=200),
+            np.linspace(30e-9, L, num=200),
+        ]
+    )
+    my_model.mesh = F.Mesh1D(vertices)
+
+    # B material parameters
+    b_density = 1.34e29  # atoms/m3
+    D_0 = 1.07e-6  # m^2/s
+    E_D = 0.3  # eV
+    boron = F.Material(
+        D_0=D_0,
+        E_D=E_D,
+        name="boron",
+    )
+
+    # mb subdomains
+    b_subdomain = F.VolumeSubdomain1D(id=1, borders=[0, L], material=boron)
+    inlet = F.SurfaceSubdomain1D(id=1, x=0)
+    outlet = F.SurfaceSubdomain1D(id=2, x=L)
+
+    my_model.subdomains = [b_subdomain, inlet, outlet]
+
+    # hydrogen species
+    mobile_D = F.Species("D")
+    mobile_T = F.Species("T")
+
+    trap1_D = F.Species("trap1_D", mobile=False)
+    trap1_T = F.Species("trap1_T", mobile=False)
+    trap2_D = F.Species("trap2_D", mobile=False)
+    trap2_T = F.Species("trap2_T", mobile=False)
+    trap3_D = F.Species("trap3_D", mobile=False)
+    trap3_T = F.Species("trap3_T", mobile=False)
+    trap4_D = F.Species("trap3_T", mobile=False)
+    trap4_T = F.Species("trap3_T", mobile=False)
+
+    # traps
+    empty_trap1 = F.ImplicitSpecies(  # implicit trap 1
+        n=6.867e-1 * b_density,
+        others=[trap1_T, trap1_D],
+        name="empty_trap1",
+    )
+
+    empty_trap2 = F.ImplicitSpecies(  # implicit trap 2
+        n=5.214e-1 * b_density,
+        others=[trap2_T, trap2_D],
+        name="empty_trap2",
+    )
+
+    empty_trap3 = F.ImplicitSpecies(
+        n=2.466e-1 * b_density,
+        others=[trap3_T, trap3_D],
+        name="empty_trap3",
+    )
+
+    empty_trap4 = F.ImplicitSpecies(
+        n=1.280e-1 * b_density,
+        others=[trap4_T, trap4_D],
+        name="empty_trap4",
+    )
+
+    my_model.species = [
+        mobile_D,
+        mobile_T,
+        trap1_D,
+        trap1_T,
+        trap2_D,
+        trap2_T,
+        trap3_D,
+        trap3_T,
+        trap4_D,
+        trap4_T,
+    ]
+
+    # hydrogen reactions - 1 per trap per species
+    interstitial_distance = 8e-10  # m
+    interstitial_sites_per_atom = 1
+
+    # TODO: fix p_0 values
+    my_model.reactions = [
+        F.Reaction(
+            k_0=D_0 / (interstitial_distance * interstitial_sites_per_atom * b_density),
+            E_k=E_D,
+            p_0=1e13,
+            E_p=1.052,
```

Comment:
we should have a reference for these values somewhere (here and/or in the docs)

Reply:
can add to docs when time comes, it's the same source as above

Reply:
There is no issue having the same comment in two places. If it is from an unpublished TDS work from Etienne, please write it